stdlib/public/Platform/android.modulemap

```diff
@@ -0,0 +1,598 @@
+// The module map for the Android NDK.
+
+// The top level 'Android' module can be included from Swift.
+module Android [system] {
+  header "SwiftAndroidNDK.h"
+  export *
+}
+
+// The modules for the Bionic C standard library used by
+// the Android NDK.
+module bionic_assert [system] {
+  // <assert.h>'s use of NDEBUG requires textual inclusion.
+  textual header "assert.h"
+  export *
+}
+module bionic_complex [system] {
+  header "complex.h"
+  export *
+}
+module bionic_ctype [system] {
+  header "ctype.h"
+  export *
+}
+module bionic_errno [system] {
+  header "errno.h"
+  export *
+}
+module bionic_fenv [system] {
+  header "fenv.h"
+  export *
+}
+// Use 'no_undeclared_includes' to avoid pulling in the libc++ headers. 
```

Comment:
This is a problem, we really need inttypes.h's include of stdint.h to get the libc++ one.

Reply:
Unfortunately I'm unable to pull in libc++'s stdint into inttypes.here, as libc++ in the NDK is still using the monolithic `std` module, before the module was split into individual `std_` modules. I added a FIXME there , as I will be able to drop this once the NDK updates it's libc++ module map. We will be able to vend a second module map then and fix this when we have a new NDK.